.github/workflows/pr-breaking-change-check.yaml

```diff
@@ -32,6 +32,8 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v4
+        with:
+          ref: ${{ github.event.pull_request.base.sha }}
```

Comment:
This isn't needed, and you should always rebase to main when updating the pr.

Reply:
Nobody is rebasing when they add a couple of commits to fix something while the main branch does main branch things. Sorry, but that's just not how things work.